docs/blogs/2023/2023-06-07-fast-conformer.md

```diff
@@ -0,0 +1,210 @@
+
+## Global Attention
+
+For global attention on modest files (upto 15-18 minutes on an A100), you can perform the following steps :
+
+```python
+from nemo.collections.asr.models import ASRModel
+  
+model = ASRModel.from_pretrained("nvidia/stt_en_fastconformer_hybrid_large_pc")
+model.transcribe(["<path to a audio file>.wav"])  # ~10-15 minutes long!
+
+```
+
+## Local Attention
+
+Coming in NeMo 1.20, you can easily modify the attention type to local attention after building the model. Then you can also apply audio chunking for the subsampling module to perform inference on huge audio files!
+
+For local attention on huge files (upto 11 hours on an A100), you can perform the following steps :
+
+```python
+from nemo.collections.asr.models import ASRModel
+  
+model = ASRModel.from_pretrained("nvidia/stt_en_fastconformer_hybrid_large_pc")
+
+model.change_attention_model("rel_pos_local_attn", [128, 128])  # local attn
+
+# (Coming in NeMo 1.20)
+model.change_subsampling_conv_chunking_factor(1)  # 1 = auto select
+
+model.transcribe(["<path to a huge audio file>.wav"])  # 10+ hours !
+
+```
+
+# Results
+
+By performing the simple modifications in Fast Conformer, we obtain strong scores throughout multiple speech tasks as shown below, all while having more efficient training and inference.
+
+For ASR alone, we obtain a 2.8x speedup as compared to Conformer encoder of similar size for inference, and can use larger batch sizes during training to further speedup training. We also compare results against tasks such as Speech Translation and Spoken Language Understanding in order to validate that these changes lead only to improvements in efficiency and do not degrade accuracy on downstream tasks.
+
+## ASR Results
+
+Below, we list some of our results on Fast Conformer on LibriSpeech test-other. We compare against the original Conformer and other recent efficient architectures. We compare against the [Efficient Conformer from Burchi2021](https://arxiv.org/abs/2109.01163) which uses a progressive downsampling of the Conformer architecture. We also compare against [Kim2022 SqueezeFormer](https://arxiv.org/abs/2206.00888) which uses a U-Net like architecture to progressively downsample the input and upsample it to 40 ms resolution prior to applying the decoder. 
+
+We find that Fast Conformer is able to achieve the same accuracy as the Conformer while being 2.8x faster and using fewer parameters. 
+
+*Fast Conformer-Large with CTC and RNNT decoders trained on Librispeech. Greedy WER (%) was measured on Librispeech test-other.  The number of parameters  (M) and compute (Multiply-Acc, GMAC) are shown for encoder only.*
+
+| Encoder                     	|   WER, %   	| Params, 	| Compute, 	|
+|-----------------------------	|:----------:	|:-------:	|:--------:	|
+|                             	| test-other 	|    M    	|   GMACS  	|
+|         RNNT decoder        	|            	|         	|          	|
```

Comment:
Table didn;t parse well here for RNNT and CTC decoder subsections. Bold it and put it in center may be. 
Also remove , on WER params and compute. Also better to merge test-other, M and GMACS to top row. Also change test-other to LS test-other

Reply:
There will be differences between markdown tables and latex ones. It is mostly ok. Good idea about bolding.

Those values are needed though to show the improvement. Bolded the headers.